### ISSUE TITLE
Text from childNodes of <li> not added during addHtml()

### ISSUE THREAD

**Issue by raf-sh** [PHPOffice/PHPWord#945]
When importing data from HTML generated by WYSWYG editor 
code usually contains a lot of nested tags this leads to having no text data in final docx document

example of html from WYSWYG editor:
```html
<ul>
	<li>
		<span style="font-family: arial,helvetica,sans-serif;">
			<span style="font-size: 12px;">
				list item1
			</span>
		</span>
	</li>
	<li>
		<span style="font-family: arial,helvetica,sans-serif;">
			<span style="font-size: 12px;">
				list item2
			</span>
		</span>
	</li>
</ul>
```

**Comment by troosan** [comment]
thanks for this (and the fix), please check the develop branch, this should be fixed now.

**Comment by troosan** [comment]
complete fix in #1239

**Comment by samimussbach** [comment]
I can confirm that your fix works with our data, Thank you!
